force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -57,8 +63,13 @@ public with sharing class FlowControlLoopsCollections {
     public static Integer findMax(Integer num1, Integer num2) {
         // Initialize largestNum as null
         Integer largestNum = null;
-
-        return null; // Replace null with the variable you used to store the result
+        if(num1 == null || num2 == null){
+            return null;
+        }
+        else if(num1 == num2){
+                return num1;
+            }
+        return Math.max(num1, num2); // Replace null with the variable you used to store the result
     }
```

Comment:
Great job using Math.max(). No need to check for equality on line 69. If they are both the same just return what ever Math.max gives you